Arelle/Arelle
# Draft: Table Linkbase Certification 2022

Reason for Change
Table Linkbase work to prepare for Certification in 2022

Description
Remove support for pre-REC versions of table linkbase (Eurofiling and pre-REC draft versions), add PWD 1.1 support.
Update table linkbase implementation to pass certification suite.
Adapt GUI to current Python support of Tcl/Tk 8.6.12.
Provide execution components as needed for cx_Frozen distribution builds.

Steps to Test
Run certification suite.  
Ensure test script results (script operation compares expected and actual Layout Model outputs) and review GUI outputs on screen and html outputs produced against expected html outputs.  
Test by script and GUI on cx_Frozen builds for Mac, Win-64, RedHat and Ubuntu.

review:
@derekgengenbacher-wf @sagesmith-wf @hermfischer-wf
